lib/styles/storybook.scss

```diff
@@ -65,3 +65,7 @@ html {
 		color: $color-default-icon;
 	}
 }
+
+.modalHeader {
```

Comment:
why we don't define default typography for modal header in `Modal.vue`?

Reply:
In modal there is slot, so there could be everything: h1, h2, h3. So I decided this CSS rule it's not a part of modal, it comes from outside. If we are using always h3 for modal title in slot, we can make exception and move this css rule to `Modal.vue` for `h3` tag.